File: java/src/baekjoon/Baek10250.java
package baekjoon;


import java.util.Scanner;

public class Baek10250 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int T = Integer.parseInt(sc.nextLine());
		for (int i = 0; i < T; i++) {
			int H = sc.nextInt();
			int W = sc.nextInt();
			int N = sc.nextInt();

			int floor = N % H;
			int number = N / H + 1;

			int room;

			if (floor == 0) {
				room = (H * 100) + number;
			} else {
				room = (floor * 100) + number;
			}
		}

	}
}

File: python/baekjoon/Baek10250.py
T = int(input())

for _ in range(0, T):
  H, W, N = map(int, input().split())

  floor = N % H;
  number = N // H;

  if floor == 0:
    print(H * 100 + number)
  else:
    print(floor * 100 + number + 1)
